### ISSUE TITLE
PLease add the projects on ai-ml here

### ISSUE THREAD

**Issue by Ayu-hack** [Ayu-hack/GeeksforGeeks-POTD-Solution#25]
Comment project name here so that I can assign  with project name and then make PR.

**Comment by praveenarjun** [comment]
# Eye Disease Classified RNN  assign me this in this repository i will PR in this repository.

**Comment by Ayu-hack** [comment]
@praveenarjun assigned.

**Comment by praveenarjun** [comment]
completed this issue

**Comment by dvamsidhar2002** [comment]
**### Breast Cancer Classification Web App using Streamlit** 
This is a machine learning project which is also deployed as a web app using Streamlit.

**Comment by praveenarjun** [comment]
I am going to add Movie Classification project ->creating a new Branch -> adding in this folder

**Comment by Ayu-hack** [comment]
@dvamsidhar2002 @praveenarjun assigned. Make PR in mini projects folder only. DONT CREATE ANY OTHER FOLDERS.

**Comment by Aishika3** [comment]
I am going to add **Text Summariser which is a streamlit project helping to summarise any document or any part of script**
@Ayu-hack add me as an assignee for adding the above.

**Comment by Ayu-hack** [comment]
@Aishika3 assigned.

**Comment by Ayu-hack** [comment]
@YashviMehta03 assigned. Please share the screenshot of working in PR

**Comment by YashviMehta03** [comment]
@Ayu-hack , yes i have created a PR and shared screenshot of working. Kindly review it.

**Comment by AdityaRoy999** [comment]
Stock prediction Dashboard using sklearn and graph libraries.

**Comment by Ayu-hack** [comment]
@AdityaRoy999 assigned. But this project already exists.

**Comment by pahalsrivastava** [comment]
Emotion Based Music Recommender (uses streamlit and keras)

**Comment by Ayu-hack** [comment]
@pahalsrivastava assigned

**Comment by dvamsidhar2002** [comment]
Hey! I want to add other ml projects which I have worked on. Can I add those projects one after the other or do I need to put a issue every time I want to add a new project? Please, clear my query.

Regards,
D Vamsidhar

**Comment by Ayu-hack** [comment]
@dvamsidhar2002 
Directly create PR of each project separately in FOLDER **PROJECT AI/ML**.
Make PRs in intervals of a few hours to **avoid spamming**.
Share screenshots of the working projects in the PRs to facilitate merging.

**Comment by dvamsidhar2002** [comment]
Sure! Ayush. Thanks for the response.

**Comment by Sumith540** [comment]
pls assign me

**Comment by Nituhimani** [comment]
Breast Cancer Classification Web App using Streamlit

**Comment by taneeshamadhu18** [comment]
accesibility tool for visually impaired,please assign me this

**Comment by Ayu-hack** [comment]
@Nituhimani @taneeshamadhu18 assigned
